server/channels/app/plugin_api_test.go

```diff
@@ -1896,6 +1896,91 @@ func TestPluginHTTPConnHijack(t *testing.T) {
 	require.Equal(t, "OK", string(body))
 }
 
+func TestPluginMFAEnforcement(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	th.App.Srv().SetLicense(model.NewTestLicense("mfa"))
+
+	pluginCode := `
+	package main
+
+	import (
```

Comment:
nit: There's some other patterns that avoid embedding the plugin code directly as a string, but allow them to be a discrete file that helps with future refactoring, etc. Not sure if we can adopt that pattern here?